timezone.gemspec

```diff
@@ -14,19 +13,23 @@ Gem::Specification.new do |s|
   s.summary     = "timezone-#{Timezone::VERSION}"
   s.license     = 'MIT'
   s.description = 'Accurate current and historical timezones for Ruby with ' \
-    'support for Geonames and Google latitude - longitude lookups.'
+                  'support for Geonames and Google latitude - longitude ' \
+                  'lookups.'
 
   s.files       = `git ls-files`.split("\n")
-  s.test_files  = `git ls-files -- {test,spec,features}/*`.split("\n")
   s.executables = `git ls-files -- bin/*`
     .split("\n").map { |f| File.basename(f) }
 
   s.extra_rdoc_files = ['README.markdown', 'License.txt']
   s.rdoc_options     = ['--charset=UTF-8']
   s.require_paths    = ['lib']
 
+  s.required_ruby_version = '>= 2.2'
```

Comment:
I am not objected to dropping 2.2 support, but it's unclear to me why it stopped working with these changes. If it's feasible to add 3.2 support independently of the RuboCop changes to see if everything still works, I would prefer not to drop 2.2 support unnecessarily.

Reply:
Given that Rubocop is being run in the CI pipeline (with a Ruby version of 3.0), I'm not sure if it's feasible.  I'm not going to invest the time to determine that, but you should feel free to take whatever you want from this PR and explore that.